app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -609,4 +633,12 @@ private void selectImages() {
 
         startActivityForResult(intent, INTENT_REQUEST_GET_IMAGES);
     }
+
+    private boolean isFileExist(String mFilePath) {
```

Comment:
Also add javadoc comments to the function (View comments on other files in FileUtils.java for reference)